packages/desktop/src/renderer/styles/index.css

```
@@ -584,17 +760,125 @@ body {
+
+/* The reassurance gets a quiet accent rule — it's good news, not small print. */
+.explain__ok {
+  border-left: 2px solid var(--accent-dim);
+  padding-left: var(--s2);
+}
+
+.explain__detail {
+  font-family: var(--font-mono);
+  font-size: 10px;
+  opacity: 0.75;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)